packages/keyring-snap-sdk/src/snap-utils.test.ts

```diff
@@ -1,4 +1,5 @@
-import { KeyringEvent } from './events';
+import { KeyringEvent } from '@metamask/keyring-api';
```

Comment:
Again, events are part of the public API since we want Snap Accounts to be able to use them with their implementation.